### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sunchao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ while let Some(record) = iter.next() {
   println!("{}", record);
 }
 ```
-See crate documentation on available API.
+See [crate documentation](https://sunchao.github.io/parquet-rs/master/parquet/) on available API.
```
Comment (sadikovi) [inline README.md]: Should this link point to gh-pages branch?

Comment (sunchao) [reply README.md]: Hmm... you mean `https://github.com/sunchao/parquet-rs/tree/gh-pages`?

Comment (sadikovi) [reply README.md]: All good. I confused it with actual code in gh-pages. Link works, no need to change.

Comment (sadikovi) [reply README.md]: I think you can actually use sunchao.github.io/parquet-rs/master. There is an index.html page with redirect.

Comment (sunchao) [reply README.md]: Ah OK. I'll change to `https://sunchao.github.io/parquet-rs/master` then.

**Comment by sadikovi** [comment]
Good to merge!

**Comment by sunchao** [comment]
Merged. Thanks @sadikovi .

**Pull request merged**